ToNote/Controls/NotePanel.cs

```diff
@@ -20,10 +20,10 @@ public NotePanel()
             // CTRL + Arrow Up or CTRL + Arrow Down navigates to an item that is above or below currently focused one, respectively.
             this.PreviewKeyDown += (s, e) =>
             {
-                if (e.Key == Key.Up && Keyboard.Modifiers == ModifierKeys.Control)
+                if ((e.Key == Key.Add || e.Key == Key.OemPlus ) && Keyboard.Modifiers == ModifierKeys.Control)
```

Comment:
Maybe it would be better to invert this? Right -> down relation seems more natural to me, right now the navigation seems a bit unnatural.
Also, update readem with updated shortcuts.